### PR TITLE
v2.16.0: Switch to ASGI by default, add Dockerfiles, ingestion refactor/improvements

### PR DESCRIPTION
**Pull request by davidlougheed**
* Refactor ingestion module into multiple files
* Allow augmenting `extra_properties` post-hoc for individuals
* Re-use phenotypic features more readily

ASGI here is still backwards compatible since I kept `wsgi.py`around.

### CODE REVIEW

**Review by brouillette — APPROVED** [review]
Works locally!
Able to update an `extra_properties` field for a an `individual` by simply reingesting the newly modified phenopackets file

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Run Bento build action
+        uses: bento-platform/bento_build_action@v0.6
```
Comment (brouillette) [inline .github/workflows/build.yml]: might we want to update `bento-platform/bento_build_action` to `v0.9.3`?

**Review by brouillette — DISMISSED** [review]
Nitpicking, but looks good otherwise

**Comment by davidlougheed** [comment]
i also fixed the image build so that it would work for the `develop` branch...

**Review by brouillette — CHANGES_REQUESTED** [review]
getting this error now
```
  File "/app/chord_metadata_service/restapi/dats_schemas.py", line 47, in _get_creators_schema
    dats_creators_schema = open(os.path.join(DATS_PATH, '{}.json'.format(creator_type)))
FileNotFoundError: [Errno 2] No such file or directory: '/app/chord_metadata_service/dats/person_schema.json'
```

**Review by brouillette — APPROVED** [review]
works for me!

**Pull request merged**